css/smacss_module.css

```diff
@@ -0,0 +1,89 @@
+.post-title{
+  font-family: 'Oswald', sans-serif;
+  font-size: 40px;
+  font-weight: bolder;
+  line-height: 4rem;
+  width: 824px;}
+
+.small-all-caps {
+  font-family: 'Abel', sans-serif;
+  font-size: 12px;
+  color: rgb(85,85,85);
+}
+
+.social-icons {
+  margin: 2px;
+}
+
+.social-icons img:nth-child(6) {
+  margin: 0 0 0 4px;
+}
+.float-right {
+  float: right;
+}
+
+.main-img {
```

Comment:
If you make the containing parent 940px then it will make all it's children that size.